.github/workflows/ci.yml

```diff
@@ -62,8 +62,21 @@ jobs:
           path: ~/.cargo/registry
           key: rust-${{ matrix.rust_toolchain }}-cargo-registry-directory-${{ hashFiles('Cargo.lock') }}-v1
 
+      - name: Cache node_modules directory
+        uses: actions/cache@v1
+        with:
+          path: waves/node_modules
+          key: node-${{ matrix.rust_toolchain }}-node-modules-directory-${{ hashFiles('waves/yarn.lock') }}-v1
```

Comment:
Why is the `rust_toolchain` part of the node cache key?

Reply:
copy pasta :)